citesphere/src/main/webapp/WEB-INF/views/auth/authorities/edit.jsp

```diff
@@ -0,0 +1,41 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<%@ taglib uri = "http://java.sun.com/jsp/jstl/functions" prefix = "fn" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<style>
+.popover {
+	min-width: 300px;
+}
+</style>
+<script>
+</script>
+
+<ol class="breadcrumb">
+  <li><a href="<c:url value="/" />">Home</a></li>
+  <li><a href="<c:url value="/auth/authority/list" />">Managed Authority Entries</a></li>
+  <li class="active">${entry.name}</li>
+</ol>
+<h2>
+	<strong><c:if test="${not empty entry.name}">${entry.name}</c:if></strong>	  
```

Comment:
if entry.name is empty than ```${entry.name}``` should should print an empty string, so the if is not necessary.